boot/freeldr/freeldr/ntldr/wlmemory.c

```diff
@@ -238,6 +239,41 @@ WinLdrSetupMemoryLayout(IN OUT PLOADER_PARAMETER_BLOCK LoaderBlock)
         return FALSE;
     }
 
+    // Always map the ACPI table if present, otherwise Windows will crash.
+    if (Extension->AcpiTable)
+    {
+        union
+        {
+            PDESCRIPTION_HEADER Header;
+            PRSDT_DESCRIPTOR Rsdt;
+            PXSDT_DESCRIPTOR Xsdt;
+        } AcpiTable = {.Header = Extension->AcpiTable};
+
+        Status = MempSetupPaging((ULONG_PTR)Extension->AcpiTable >> MM_PAGE_SHIFT, 
+                                 MM_SIZE_TO_PAGES(Extension->AcpiTableSize),
+                                 FALSE);
```

Comment:
Should this one stay MempSetupPaging, or would a MmMarkPagesInLookupTable call similar to the one made below actually work?

Reply:
The `MmMarkPagesInLookupTable` does nothing with paging in my tests.

Reply:
So it makes the windows ACPI driver crash?

Reply:
the Windows kernel crash.